final_task/rss_reader/add_to_csv.py

```diff
@@ -0,0 +1,17 @@
+import csv
+
+def addcsv(articles):
+    with open('news.csv', 'a', newline='') as csvfile:
+        fieldnames = ['link','title', 'img', 'summary', 'published']
+
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+
+        writer.writerow(articles)
```

Comment:
As far as I understand if you will get news 5 times from the same rss source, then you will have these news 5 times repeated in your file